src/components/content-module/content-module.tsx

```diff
@@ -1,24 +1,59 @@
 import * as React from 'react';
 import { ContentModule } from '../../constants';
 
-export const ModuleComponent = ({module}: {module: ContentModule}) => (
-  <article className="center mw5 mw6-ns br3 hidden ba b--black-10 mv4 tc">
+export const ModuleComponent = ({module}: {module: ContentModule}) => {
+
+  const displayDependenciesIfPossible = (mod) => {
+    if (mod.dependencies && mod.dependencies.length) {
```

Comment:
**nit**: These conditions could be shortened to ternaries, as long as the expression returns an explicit `Boolean` 🙂

Check out [`home.component.tsx`](https://github.com/bridge-school/content-dashboard/blob/master/src/scenes/home/home.component.tsx#L32) for an example implementation!